### DISCUSSION TITLE
WhatsApp archive

### DISCUSSION THREAD

**Discussion by Illaise** [devlikeapro/waha#214] [General]
Is it possible to perform actions on a WhatsApp archive using this API? For example, can we archive chats or, if we are doing a newsletter, can we ignore archived chats?

**Comment by allburov** [comment]
right now it's possible only to remove chat. https://waha.devlike.pro/docs/how-to/chats/#delete-chat
You can create an issue and we'll add that feature as well, it's probably possible in WEBJS engine (default one)!

**Comment by devlikepro** [comment]
Available in [2024.8.1](https://waha.devlike.pro/blog/waha-2024.8-whatsapp-business-labels-api-chat-archive-and-more/)

----
👉 Read more about the endpoints on [💬 Chats](https://waha.devlike.pro/docs/how-to/chats/#archive-chat) page!

### Archive chat

Use the method to archive chat
```bash
POST /api/{session}/chats/{chatId}/archive
```

### Unarchive chat

Use the method to unarchive chat

```bash
POST /api/{session}/chats/{chatId}/unarchive
```

### chat.archive
The `chat.archive` event is triggered when a chat is archived or unarchived.
```json
{
  "event": "chat.archive",
  "session": "default",
  "payload": {
    "id": "123123123@c.us",
    "timestamp": 1667561485,
    "archived": true <== or false
  },
  ...
}
```


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
